dev-envs/linux/ssh.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -l
+IFS=$'\n\t'
+set -euxo pipefail
+
+# New versions require OpenSSL >= 1.1.1
+VERSION="V_9_3_P2"
+url="https://github.com/openssh/openssh-portable/archive/refs/tags/${VERSION}.tar.gz"
+
+archive_name=$(basename "${url}")
+workdir="/tmp/setup-${archive_name}"
+mkdir -p "${workdir}"
+curl "${url}" -Lo "${workdir}/${archive_name}"
+tar -xf "${workdir}/${archive_name}" -C "${workdir}" --strip-components 1
+
+useradd -r -s /sbin/nologin sshd
+
+pushd "${workdir}"
+autoreconf
+./configure
+make -j "$(nproc)"
+make install
+popd
+rm -rf "${workdir}"
+
+# Ameliorate transient network issues for things that can
+# retry like Visual Studio Code's Remote - SSH extension
+passwd -d root
+cat <<'EOF' >> /usr/local/etc/sshd_config
+PermitRootLogin yes
+PermitEmptyPasswords yes
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
An alternative to this is setting up authenticated_keys passing a public key at devenv creation. This way when you first create an instance of the container, it is setup to accept your ssh key connection

Reply:
What is the benefit if it's on your local machine or in the cloud behind AppGate?